src/main/java/com/flippingutilities/ui/widgets/chart/PriceRange.java

```diff
@@ -0,0 +1,15 @@
+package com.flippingutilities.ui.widgets.chart;
+
+public final class PriceRange {
```

Comment:
missing some javadoc to explain what a price range is, we need to infer it from its usage at the moment